cg/services/fastq_file_service/utils.py

```diff
@@ -28,19 +30,43 @@ def concatenate_reverse_reads(directory: Path) -> Path | None:
 
 def get_new_unique_file(directory: Path) -> Path:
     unique_id = uuid.uuid4()
-    return Path(directory, f"{unique_id}.fastq.gz")
+    return Path(directory, f"{unique_id}{FileExtensions.FASTQ}{FileExtensions.GZIP}")
 
 
 def get_forward_read_fastqs(fastq_directory: Path) -> list[Path]:
-    return get_fastqs_by_direction(fastq_directory=fastq_directory, direction=1)
+    return get_fastqs_by_direction(fastq_directory=fastq_directory, direction=ReadDirection.FORWARD)
+
+
+def get_concatenated_read_output_path(
```

Comment:
Consider moving this logic to another module - we don't want to spread microbial pipeline logic into this module.

Reply:
This is not microbial pipeline logic

Reply:
It is how we want to deliver all concatenated files to be named

Reply:
Alright, lets keep it here then 👍 
From a design perspective, it might better to separate it into another service to keep the concerns separate. We can extract it if other naming standards are requested in the future

Reply:
I agree! We can make the name better though as this is meant solely for delivery. @henrikstranneheim commented on that earlier as well!